### DISCUSSION TITLE
Would it be possible for OTVDM/WineVDM to be used on Windows 3.1 games that could be sold on Steam or GOG?

### DISCUSSION THREAD

**Discussion by Wadmodder** [otya128/winevdm#1431] [Q&A]
I wanted to write this discussion regarding any possibilities of game developers and publishers using the OTVDM/WineVDM project for the 64-bit versions of Microsoft's Windows NT operating systems for re-releasing their older Windows 3.1 and earlier PC games made in the early to mid 1990s on Steam and GOG, and what the legality of the use of your OTVDM has for 16-bit Windows PC games and game preservation in general.

As Microsoft discontinued the NTVDM and the 16-bit Windows on Windows subsystems with the release of Windows 11, as it dropped support for all 32-bit processors, I could see OTVDM having some potential for re-releasing old 16-bit Windows 3.1 and earlier PC games on Steam and GOG, in a similar vein to how DOSBox has been used by PC game developers and publishers to re-release their older MS-DOS games on those platforms.

Hopefully it would be possible for any PC game publishers to use the OTVDM to re-release many old PC games designed for Windows 3.1 and earlier, which are currently unavailable on either Steam and GOG.

**Comment by cracyc** [comment]
Sure, the project is GPL but you'd need to ask the actual right holders and they probably aren't looking here.
